packages/legacy/core/App/components/misc/CredentialCard.tsx

```diff
@@ -1,12 +1,11 @@
 import { CredentialExchangeRecord } from '@aries-framework/core'
+import { Attribute, BrandingOverlayType, Predicate } from '@hyperledger/aries-oca/build/legacy'
```

Comment:
Ok for now but eventually let's refactor this to not reference from `/build`

Reply:
Thats the idea. Using `legacy.` in every reference will make it harder to refactor once the legacy tipes are gone. I don't like the `build` folder either, but since it won't be definitive, I think it's ok for now.